src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -92,6 +98,33 @@ def load_arguments(self, _):
         c.argument('soft_delete_feature_state', arg_type=get_enum_type(['Enable', 'Disable']), help='Set soft-delete feature state for a Recovery Services Vault.')
         c.argument('cross_region_restore_flag', arg_type=get_enum_type(['True', 'False']), help='Set cross-region-restore feature state for a Recovery Services Vault. Default: False.')
 
+    # Identity
+    with self.argument_context('backup vault identity assign') as c:
+        c.argument('vault_name', vault_name_type)
+        c.argument('system_assigned', action='store_true', options_list=['--system-assigned'], help="Provide this flag to enable system assigned identity for Recovery Services Vault.")
+        c.argument('user_assigned', user_assigned_type)
+
+    with self.argument_context('backup vault identity remove') as c:
+        c.argument('vault_name', vault_name_type)
+        c.argument('system_assigned', action='store_true', options_list=['--system-assigned'], help="Provide this flag to remove system assigned identity for Recovery Services Vault.")
+        c.argument('user_assigned', user_assigned_remove_type)
+
+    with self.argument_context('backup vault identity show') as c:
+        c.argument('vault_name', vault_name_type)
+
+    # Encryption
+    with self.argument_context('backup vault encryption') as c:
+        c.argument('vault_name', vault_name_type)
+
+    with self.argument_context('backup vault encryption update') as c:
+        c.argument('encryption_key_id', encryption_key_id_type)
+        c.argument('infrastructure_encryption', infrastructure_encryption_type)
+        c.argument('identity_id', user_assigned_identity_id_type)
+        c.argument('use_systemassigned_identity', action='store_true', options_list=['--use-system-assigned'], help="Provide this flag to use system assigned identity for encryption.")
```

Comment:
Could we refer to the recommendation in the [guideline.md#enable-managed-identity-during-resource-creation](https://github.com/Azure/azure-cli/blob/dev/doc/managed_identity_command_guideline.md#enable-managed-identity-during-resource-creation), change the parameter name to `--mi-system-assigned`

Reply:
@zhoxing-ms : we don't have creation time experience for managed identities in the vault. This parameter is used for Customer managed key encryption properties for Vault

Reply:
Same as above